mdsuite/graph_modules/molecular_graph.py

```diff
@@ -134,9 +144,13 @@ def build_configuration_graph(self, cutoff: float, adjacency: bool = True):
         -------
 
         """
-        path_list = [join_path(species, 'Positions') for species in self.species]
-        configuration_tensor = tf.concat(self.database.load_data(path_list=path_list, select_slice=np.s_[:, 0]), axis=0)
-        distance_matrix = self.get_neighbour_list(configuration_tensor, cell=self.experiment.box_array)
+        path_list = [join_path(species, 'Positions') for species in
+                     self.species]
+        configuration_tensor = tf.concat(
+            self.database.load_data(path_list=path_list,
+                                    select_slice=np.s_[:, 0]), axis=0)
+        distance_matrix = self.get_neighbour_list(configuration_tensor,
+                                                  cell=self.experiment.box_array)
```

Comment:
This would be more reabable I assume

```py
configuration_tensor = tf.concat(
            self.database.load_data(path_list=path_list, select_slice=np.s_[:, 0]),
            axis=0
        )
distance_matrix = self.get_neighbour_list(configuration_tensor, cell=self.experiment.box_array)
```